draft-thomassen-dnsop-generalized-dns-notify.md

```diff
@@ -408,32 +436,27 @@ Class
 
 RRtype
         The type of generalised NOTIFY that this NOTIFY RR defines
-		the desired target address for. Currently only the types CDS,
+        the desired target address for. Currently only the types CDS,
         CSYNC and DNSKEY are suggested to be supported, but there is
-		no protocol issue should a use case for additional types of
-		notifications arise in the future.
+        no protocol issue should a use case for additional types of
+        notifications arise in the future.
  		
 Scheme
-	    The scheme for locating the desired notification address.
+        The scheme for locating the desired notification address.
         The range is 0-255. This is a 16 bit unsigned integer in
-		network byte order. The value 0 is an error. The value 1 is
-		described in this document and all other values are currently
-		unspecified.
+        network byte order. The value 0 is an error. The value 1 is
+        described in this document and all other values are currently
+        unspecified.
 
 Port
-	    The port on the target host of the notification service. The
+        The port on the target host of the notification service. The
         range is 0-65535. This is a 16 bit unsigned integer in network
-		byte order.
+        byte order.
 
 Target
         The domain name of the target host providing the service of
         listening for generalised notifications of the specified type.
-        There MUST be one or more address records for this name, the
-        name MUST NOT be an alias (in the sense of RFC 1034 or RFC
-        2181). 
-
-        A Target of "." means that the service is decidedly not
-        available at this domain.
+        There MUST be one or more address records for this name. 
```

Comment:
This wording also may be read to preclude aliases. Suggestion: `This name MUST resolve to one or more address records.`

Reply:
Hmm. I see your point and agree with your suggestion.

Reply:
done